### DISCUSSION TITLE
Allow the creation of submenus for `<Menu>`

### DISCUSSION THREAD

**Discussion by hornta** [mantinedev/mantine#2128] [Feature requests]
Example: https://codesandbox.io/s/admiring-lamport-5wt3yg?file=/src/App.tsx

![image](https://user-images.githubusercontent.com/4553604/184594243-2a2d7014-0b58-4b40-9ddb-aef2db5d13c6.png)

**Comment by rtivital** [comment]
It is in the backlog, I cannot tell when it will be added at this moment

**Comment by qu1ck** [comment]
I would love to see submenu functionality. Having to resort to hacky solutions to cobble together this functionality using current menu component is not fun.
